.gitignore

```diff
@@ -38,7 +38,7 @@ pip-delete-this-directory.txt
 npm-debug.log*
 
 # App specific
-.influencetx-secrets.env
+env.sh
```

Comment:
You might keep the `.influencetx-secrets.env` line in the gitignore, too, so that people with the previous configuration who have the old file hanging around don't accidentally commit it after the gitignore changes.